src/main/kotlin/net/atos/zac/signalering/SignaleringenJob.kt

```diff
@@ -67,7 +67,7 @@ class SignaleringenJob @Inject constructor(
      */
     fun zaakSignaleringenVerzenden() {
         val signaleringVerzendInfo = SignaleringVerzendInfo()
-        LOG.info("Zaak signaleringen verzenden: gestart...")
+        LOG.fine("Zaak signaleringen verzenden: gestart...")
```

Comment:
I switched this from `FINE` to `INFO` log level recently because I think we want this to be logged by default for two reasons:
- we also log at INFO level in `taakSignaleringenVerzenden` and it is very confusing /misleading not to do the same here
- this code is only called once per day from our cronjob and actually I want to be able to check this in the logs. I think this is a good example of where we do want to log this by default (INFO log level)

Reply:
Interesting topic, won't go into this too much, but my 2 cent here:
- I'd say we should log the actual Job starting, so that should tell us this will be invoked.
- And this should be part of an OTEL trace (standard behaviour AFAIK)
- Maybe it's useful to add some logging in the Job to show it has finished and what was the result of the Job. That would mean information has to be returned on what was done.

For now - I'd say keep it the same as the `taak` functionality per suggestion.

Reply:
Not an intentional change. Got this back to info